public/admin/config.yml

```diff
@@ -154,3 +154,66 @@ collections:
                 fields:
                   - { label: Color, name: color, widget: string }
                   - { label: Content, name: content, widget: markdown }
+  - name: learn
+    label: Learn
+    format: json
+    description: Learn content
+    slug: '{{slug}}'
+    editor:
+      preview: true
+    files:
+      - name: Learn-Landing
+        label: 'Learn: Landing page'
+        file: src/cms-content/learn/learn-landing.json
+        fields:
+          - label: Header
+            name: header
+            widget: string
+            default: true
+          - label: Sections
+            name: sections
+            widget: list
+            fields:
+              - { label: Section title, name: sectionTitle, widget: string }
+              - { label: Section ID, name: sectionId, widget: string }
+              - { label: Description, name: description, widget: markdown }
+              - { label: Button CTA, name: buttonCta, widget: string }
+              - { label: Button redirect, name: buttonRedirect, widget: text }
```

Comment:
The home page seems to have only one link per section, and they are all links as far as I can tell

Reply:
yeah, my thought was that each button will need its own redirect. so for the example screenshot'ed below, the redirect will be '/glossary'

<img width="217" alt="Screen Shot 2020-10-19 at 12 31 18 PM" src="https://user-images.githubusercontent.com/44076375/96479582-27fe9980-1207-11eb-9593-8ae546776584.png">

Reply:
do you see it as needing a different setup?